### PR TITLE
Bulk variant .json file upload

### PR DESCRIPTION
**Pull request by bbunce**

### CODE REVIEW

File: vcfapp/forms.py [hunk 1]
```diff
@@ -1,5 +1,10 @@
+import json
+from json.decoder import JSONDecodeError
 from django import forms
 from django.core.exceptions import ValidationError
+from .utils import VcfAppUtils
+
+helper = VcfAppUtils()
```
Comment (rklocke) [inline vcfapp/forms.py:7]: This is smart

File: vcfapp/urls.py
Comment (rklocke) [inline vcfapp/urls.py:1]: Pls remove this file as we are handling urls in the main project urls.py

File: vcfapp/views.py [hunk 2]
```diff
@@ -1,15 +1,19 @@
 from django.core.paginator import Paginator
 from django.http import HttpResponseRedirect, Http404
 from django.urls import reverse
-from pymongo import MongoClient
-from bson import ObjectId, regex
 from django.shortcuts import render, redirect
-from .forms import SingleVariantForm
+from bson import ObjectId, regex
+from pymongo import MongoClient
```
Comment (rklocke) [inline vcfapp/views.py]: Not needed in this script as used in utils.py now

File: vcfapp/forms.py (hunk 1 above)
Comment (bbunce) [reply vcfapp/forms.py:7]: Thanks! :)

File: vcfapp/views.py (hunk 2 above)
Comment (bbunce) [reply vcfapp/views.py]: Done

File: vcfapp/urls.py
Comment (bbunce) [reply vcfapp/urls.py:1]: Done

**Review by RSWilson1 — COMMENTED** [review]
Looks good!

**Pull request merged**